erpnext/accounts/doctype/bank_account/bank_account.py

```diff
@@ -31,3 +31,10 @@ def make_bank_account(doctype, docname):
 	doc.is_default = 1
 
 	return doc
+
+
+@frappe.whitelist()
+def get_bank_account(bank_account):
```

Comment:
this is really confusing. how about `get_account_for_bank_ac`?

Reply:
Fixed!

Reply:
Hi, looks like you've forgotten to fix this..

Reply:
Did this def get_bank_account(name):

Reply:
Removed that function